src/test/java/com/wcc/platform/factories/SetupEventFactories.java

```diff
@@ -0,0 +1,23 @@
+package com.wcc.platform.factories;
+
+import com.wcc.platform.domain.cms.attributes.ProgramType;
+import com.wcc.platform.domain.cms.attributes.SimpleLink;
+import com.wcc.platform.domain.platform.Event;
+
+/** Event test factories. */
+public class SetupEventFactories {
```

Comment:
sorry, I also create factories, basically you can remove this or copy my changes to yours factory and delete the other one file